app/controllers/dradis/plugins/csv/upload_controller.rb

```diff
@@ -2,10 +2,23 @@ module Dradis::Plugins::CSV
   class UploadController < ::AuthenticatedController
     include ProjectScoped
 
+    before_action :load_attachment, only: [:new]
+
     def new
+      @default_columns = ['Unique Identifier', 'Column Header From File', 'Type', 'Field in Dradis']
```

Comment:
why not `ID`? Otherwise the column is ginormous! 

`Column Header` is definitely enough info, no?

`Type` I don't love. Entity? Model?

`Dradis Field`